mm/kfence/core.c

```diff
@@ -276,7 +276,6 @@ static void *kfence_guarded_alloc(struct kmem_cache *cache, size_t size, gfp_t g
 		/* Randomly inject "faults" by protecting the allocated object. */
 		kfence_protect(meta->addr);
 	}
-
```

Comment:
No, I accidentally deleted it when adding/deleting pr_debug calls. Fixed.
Also removed the braces around kfence_protect() above.

Reply:
I recently had this comment: https://lkml.kernel.org/r/20200721140929.GB10769@hirez.programming.kicks-ass.net -- after which I did add {} where technically it wasn't needed.

Peter has a point: if, whatever comes after the 'if (' isn't just on 1 line (or 2, if generous), add "bucket of braces". Otherwise it might hurt readability.

I leave it to you. I'm indifferent, but am trying to refine the style to follow kernel's best-practices.

Reply:
I think moving the comment before the if-statement (which I also did) solves this problem.
I agree that having more than one line without the braces may worsen the readability.

Reply:
Although the comment only applies if the config option is set. We could shorten the comment to

```
/* Random "faults" by protecting the object. */
```

And place it on the same line as kfence_protect().

Sorry for the bike-shedding. ;-)